src/main/java/homework9/GmailTests.java

```diff
@@ -0,0 +1,91 @@
+package homework9;
+
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.apache.hc.core5.reactor.Command;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class GmailTests {
+
+    private WebDriver driver;
+    private WebDriverWait wait;
+
+    @BeforeClass
+    @Parameters("browserName")
+    public void setupBrowser(String browserName) {
+        if (browserName.equals("chrome")) {
+            WebDriverManager.chromedriver().setup();
+            driver = new ChromeDriver();
+        } else if (browserName.equals("edge")) {
+            WebDriverManager.edgedriver().setup();
+            driver = new EdgeDriver();
+        } else throw new IllegalArgumentException("You have passed invalid browser name");
+
+        wait = new WebDriverWait(driver, 15);
+
+        logIn();
+    }
+
+    public void logIn() {
+        driver.get("https://mail.google.com/");
+
+        WebElement email = driver.findElement(By.xpath("//input[@type='email']"));
+        email.sendKeys("test.kliuchkovska@gmail.com");
+        email.sendKeys(Keys.ENTER);
+
+        wait.until(d -> driver.findElement(By.xpath("//input[@type='password']")).isDisplayed());
+
+        WebElement password = driver.findElement(By.xpath("//input[@type='password']"));
+        password.sendKeys("Kliuch21");
+        password.sendKeys(Keys.ENTER);
+
+        wait.until(d -> driver.findElement(By.xpath("//input[@class='gb_gf']")).isDisplayed());
+    }
+
+    @Test()
+    public void isSearchResultNotEmpty() {
+        WebElement search = driver.findElement(By.xpath("//input[@class='gb_gf']"));
+
+        search.sendKeys("Hello");
+        search.sendKeys(Keys.ENTER);
+
+        wait.until(d -> driver.findElement(By.xpath("//div[@role='main']//tr[@role='row']")).isDisplayed());
+        List<WebElement> searchResult = driver.findElements(By.xpath("//div[@role='main']//tr[@role='row']"));
+        Assert.assertNotEquals(searchResult.size(), 0);
```

Comment:
You may add messages in the asserts that will indicate what is wrong when it fail, e.g.:
```suggestion
        Assert.assertNotEquals(searchResult.size(), 0, "Search results is empty");
```
But that's not an issue